### ISSUE TITLE
download menu does not pop in firefox 112.0 64bit

### ISSUE THREAD

**Issue by jgbi** [downthemall/downthemall#444]
**Desktop (please complete the following information):**
Edition	Windows 10 Pro
Version	22H2
OS build	19045.2846
Experience	Windows Feature Experience Pack 120.2212.4190.0

 - Browser and version: firefox 112.0 64bt
 - DownThemAll! version: Version 4.8.1

**Describe the bug**
do not know if this is a defect or a change that happen because of the rewrite but below is the description of it:
In the past when a link on a website was press to download a file, the download would pop pre populated now I do not even know how to start a download using this tool, 

**To Reproduce**
install firefox extension
select a file to download
nothing happens

**Expected behavior**
downloadthemall menu should pop up populated with the file name waiting for the user to press start

**Comment by nmaier** [comment]
How are you trying to select a file exactly?

**Comment by eejeel** [comment]
I have the same issue
FF 113.0.2 with Dta V4.10.1 20230524
Windows 8.1
Problem is also in LibreWolf 113.0.2-1

If you click on a download link (for example a zip file), FF gives standard a pop-up and ask you what to do. In case of a zip file: Open with ... or Save.
With Dta installed in FF it's the same. In the FF download pop-up there is no possibility to chose for Dta or DtaOneClick.
I also have Waterfox Classic with Dta-legacy. In this case in the pop-up you can chose for Dta or DtaOneClick (and also the standard Open with ... or Save).


Why do I need this Dta or DtaOneClick possibility in the pop-up?
There are web shops where you can download the program you bought. These websites give you a download link with a hash in the link instead of the real download location/file name.
Example: https://www.example.com/download/link/index/hash/MC43MTY3MTkwMCAxNjA345zNzcxMTQ3MTIyMjM2OTM2Ng,,/

In the legacy version, the "translation" between hash and real file name is done in the browser himself (I think), because in Dta manager only the real file name is shown.
In the current version of Dta the stored file keeps the name "MC43MTY3MTkwMCAxNjA345zNzcxMTQ3MTIyMjM2OTM2Ng,," when you use the right mouse click on the link.
If you download more files at the same time, you don't know the real name of the downloaded files; it keeps the hash name.

Current version:
![No Dta in pop-up in V4 10](https://github.com/downthemall/downthemall/assets/30604538/bdbfe9e6-6cb7-4bb4-9ff5-d051e908d7b6)

Legacy version:
![pop-up in Dta legacy](https://github.com/downthemall/downthemall/assets/30604538/26950f3f-c57f-44b3-b875-7aa90ba0e4c7)

**Comment by eejeel** [comment]
@d7415 Yes, what is written in that the topic is my problem. Maybe my message can be moved to that topic. I don't know if possible by a moderator.

About the problem: what I read there is that at this moment it cannot been solved. Then Dta is not usable for me in an actual Firefox browser because the download link with a hash as written above.
I have to continue with an old Waterfox-classic and Dta Legacy version.
